fastlane/Fastfile

```diff
@@ -80,25 +80,28 @@ platform :ios do
   desc "Tag in git and push to GitHub"
   private_lane :add_tag do |options|
     next_version = options[:version]
+    next_tag = "v#{next_version}"
 
-    add_git_tag(tag: next_version)
-    push_git_tags(tag: next_version)
+    add_git_tag(tag: next_tag)
+    push_git_tags(tag: next_tag)
   end
 
 
   desc "Release pods"
   private_lane :release_pods do
     AmplifyPods.pods.each do |pod|
+      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
 
       UI.message("Pushing pod #{pod[:spec]}")
-      sh('pod', 'trunk', 'push', pod[:spec], '--allow-warnings', '--synchronous')
+
+      sh('pod', 'trunk', 'push', pod_path, '--allow-warnings', '--synchronous')
 
       # TODO: utilize action once the PR for adding `synchronous` options has been merged
       # https://github.com/fastlane/fastlane/pull/16698
       # pod_push(path: pod[:spec], allow_warnings: true, synchronous: true)
 
```

Comment:
Should be able to remove this now right?

Reply:
I kept that in, so that once the fastlane team merges my [PR](https://github.com/fastlane/fastlane/pull/16698) we can use the [fastlane action `pod_push`](https://github.com/aws-amplify/amplify-ios/pull/600/files#diff-56c5021ef06ac6cea807ae53269b6f4fR101) instead of calling the [sh command on line 97](https://github.com/aws-amplify/amplify-ios/pull/600/files#diff-56c5021ef06ac6cea807ae53269b6f4fR97)

It'll have the same effect, but just cleaner and more organized.